sources/packages/web/src/services/http/ApplicationApi.ts

```diff
@@ -31,6 +32,14 @@ export class ApplicationApi extends HttpBaseClient {
     );
   }
 
+  async getApplication(
```

Comment:
For the same endpoint, we do not need a different `ApplicationApi` method.

It means, the existing `getApplicationData()` can be re-used by the service `ApplicationService` method `getApplication()` which is newly created. Like this using the overloads. 

API:

![image](https://github.com/user-attachments/assets/2c4c6565-fedd-4e2f-84f0-fefad9c0f070)


Service: 

![image](https://github.com/user-attachments/assets/22646f40-c009-4eff-8555-3a74422afba2)

Reply:
updated